packages/configs/ts/base.json

```diff
@@ -9,7 +9,7 @@
     "isolatedModules": true,
     "useDefineForClassFields": true,
     "lib": ["es2022", "DOM", "DOM.Iterable"],
-    "module": "Preserve",
+    "module": "esnext",
```

Comment:
Can you explain this change? cc @versecafe

Reply:
@benjreinhart there shouldn't be a huge difference between esnext and preserve but removing preserve allows for some small transformations in tsc for cleaning things up, this should be a fine change and even make things easier but does mean a test to make sure the full bundled version is resolving correctly needs to be done to prevent a mishap like the 0.0.1 release bug we had

Reply:
Sweet, so long as it's all working with no major downsides LGTM

Reply:
Thank you @versecafe for the confirmation. I've discovered that "module: preserve" doesn't work well with vite-node watch mode. If necessary, we can consider keeping "module: esnext" specifically for the development mode @benjreinhart